scripts/99-img_check.sh

```diff
@@ -4,19 +4,30 @@
 # © 2018 DigitalOcean LLC.
```

Comment:
This file is not executable. Is that a problem?

Reply:
No not a problem, i did not make it executable. Should I make it executable and add it again?